client-src/overlay.js

```diff
@@ -23,10 +23,26 @@ let iframeContainerElement;
 let containerElement;
 /** @type {Array<(element: HTMLDivElement) => void>} */
 let onLoadQueue = [];
+/** @type {any} */
+let overlayTrustedTypesPolicy;
+/** @type {any} */
 
 ansiHTML.setColors(colors);
 
-function createContainer() {
+/**
+ * @param {string | null} policyName
+ */
+function createContainer(policyName) {
+  // Enable Trusted Types if they are available in the current browser.
+  if (window.trustedTypes) {
```

Comment:
We could optionally check whether the `policyName` is not `null`. However, I think using trustedTypes whenever they are available is better because:

1. It is still noop and backwards compatible change for applications which do not enforce Trusted Types
2. This enables Trusted Types integration without the need to change anything in webpack configuration (which can be hidden, such as in [CRA](https://github.com/facebook/create-react-app)). This means that the only change the application needs to do is whitelist the default policy name.